packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -328,6 +331,9 @@ export class Dialog
     if (changes.has("modal") && (this.hasUpdated || this.modal !== false)) {
       this.updateOverflowHiddenClass();
     }
+    if (changes.has("modal") || this.focusTrapDisabled) {
```

Comment:
@Elijbet this still needs to check for changes to `focusTrapDisabled`. this part `this.focusTrapDisabled` should be `changes.has("focusTrapDisabled")`

Reply:
No need to look for changes to `open` since `onOpen` should cover that.

Reply:
```suggestion
    if ((changes.has("modal") || changes.has("focusTrapDisabled")) && this.hasUpdated) {
```

Reply:
whoops, sorry typo